platforms/intellij/compose/playground/build.gradle.kts

```diff
@@ -47,6 +49,14 @@ kotlin {
         implementation(projects.platforms.intellij.compose)
       }
     }
+    jvmTest {
+      dependencies {
+        implementation(libs.junit)
+        implementation(libs.roborazzi)
+        implementation(compose.desktop.common)
```

Comment:
I added compose dependencies to the jvmTest. Before I added these dependencies, I had a `java.lang.NoClassDefFoundError`.